### PR TITLE
marco checking pull request

### PR DESCRIPTION
**Pull request by mprizzuto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 [README.md](https://github.com/mprizzuto/alpha-one-homepage/files/7018079/README.md)
 
 ## hello !
+## Perpetual Education is the best school ever!!!
```
Comment (perpetual-education) [inline README.md]: Very smart!

**Review by perpetual-education — CHANGES_REQUESTED** [review]
Just fix up that heading, Marco! And we're good to go!

**Comment by sheriffderek** [comment]
Did this get resolved, Marco?

**Comment by perpetual-education** [comment]
Looks like there are some conflicts here ???

**Comment by mprizzuto** [comment]
I dont see the conflicts, but i do see 7 issues. Where do i view the
conflicts? also, as mentioned in slack, im having internet issues today
(very slow). So may be slow in replying today

-Marco

**Comment by mprizzuto** [comment]
im still not seeing the merge conflics. I tried using this to find where to
go
https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

I see alex committing lots of things, I'll ask him

**Pull request merged**